css/store.css

```diff
@@ -0,0 +1,221 @@
+
+
+
+
+:root {
+    --grey-dark: #666666;
```

Comment:
Well done. Just noticed that you declared variables for storing colours but you keep to use hex code, but anyway it doesn't affect the performance of the page.
it's a good and strong job.